acra/configuring-maintaining/logging/improvements.md

```diff
@@ -0,0 +1,10 @@
+---
+weight: 3
+title: Improve Logging
+bookCollapseSection: true
+---
+
+# Implementing a new logging format
+
+If you think that Acra is missing some handy logging formats, feel free to open an [Issue](https://github.com/cossacklabs/acra/issues) and describe why you think it's important. Alternatively, you can add the required log formatter ([logging folder](https://github.com/cossacklabs/acra-Q12021/tree/master/logging) is a good place to start) and open a [Pull Request](https://github.com/cossacklabs/acra/pulls).<br/>
+We are grateful for contributions.  
```

Comment:
I was thinking more about just moving this to the page with log format examples. Right after the blurb but before the sections.

```markdown
{{< hint info >}}
If you believe that Acra is missing some handy logging formats,
feel free to [open an Issue](https://github.com/cossacklabs/acra/issues) and tell us.

Alternatively, you can add the required log formatter
([`logging` folder](https://github.com/cossacklabs/acra-Q12021/tree/master/logging) is a good place to start)
and [open a pull request](https://github.com/cossacklabs/acra/pulls).

We will be grateful for contributions ❤️
{{< /hint >}}
```

I've also tweaked wording a bit so that it does not look like we're accusing the reader of even thinking about adding any other format, requiring them to convince us in their usefulness.